behat.yml.dist

```diff
@@ -1,4 +1,4 @@
 default:
   gherkin:
     filters:
-      tags: ~@php8
+      tags: ~@php8&&~@cucumber-parser
```

Comment:
We should probably handle that with 2 suites running the same scenarios with each parser instead of having only a few scenarios running with the new parser. The tag would then be used only to tag scenarios that _require_ the cucumber parser

Reply:
Hm how about a second profile too?

`default` -  runs `~@php8&&~@cucumber-parser`
`cucumber` - includes `@cucumber-parser` and also enables the config setting?

Reply:
yeah, a profile rather than a suite might do it.